src/main/java/seedu/address/logic/parser/ScheduleCommandParser.java

```diff
@@ -43,22 +43,22 @@ public ScheduleCommand parse(String args) throws ParseException {
 
         argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_LOCATION, PREFIX_START_TIME, PREFIX_END_TIME);
 
-        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("h:mm a, d MMMM yyyy");
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd hhmm");
```

Comment:
Formatting should be `DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm");` instead